public/js/globals.js

```diff
@@ -0,0 +1,7 @@
+// TO DO: cookieStore IS NOT AVAILABLE ON SAFARI OR FIREFOX
+export const cookieStore = window.cookieStore;
+export const d3 = window.d3;
```

Comment:
all those symbols are attached to window by loading their files as script instead of module. we reexport them here so they appear clean in the modules